views/partials/account-sidebar.ejs

```diff
@@ -2,9 +2,14 @@
   <div class="profile-basic-info account-first-row">
     <img
       class="profile-img"
-      src="https://res.cloudinary.com/dyguovdbc/image/upload/v1676908287/pixelify/placeholder-image_ykgw2b.jpg"
+      src="<%-profileData.profileImg.thumbnail%>"
+      onerror="this.onerror=null; this.src='<%-defaultProfileImg%>'"
     />
+    <% if(!profileData.displayName) {%>
```

Comment:
I think this would be simpler and bit more easier on the eyes:
```
    <div class="profile-name"><%-profileData?.displayName || profileData?.username %></div>
```

Reply:
This definitely looks much better. I should get used to using ternary operators more often.

Reply:
It is not a ternary operator actually. It is called [Logical OR](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Logical_OR). It's really good for cases like this when you have two variables here and you expect one of them to be falsy. It evaluates the value on the left, if it's true, then it's used otherwises the value on the right is used.